django-server/server/home/templates/home/home.html

```diff
@@ -40,14 +54,55 @@
             </nav>
         {% endblock %}
     </div>
-    <div id="main-content" class="container">
+    <div id="main-content">
         {% block content %}
-            <h1>{% trans 'Welcome to EatersLab by PoohLab.' %}</h1>
+            <header id="logo-container" class="container-fluid">
+                <img src="{% static 'icon.svg' %}" alt="logo">
+                <h1>{% trans 'EatersLab' %}</h1>
+                <p>{% trans 'Cafeteria data management' %}</p>
+            </header>
+            
+            <section id="pros" class="container">
+                <div class="row">
+                    <div class="col">
+                        <h2>{% trans 'For customers' %}</h2>
+                        <p>{% trans 'Have you ever went for lunch only to find out that there are no free seats? Now you can check everything before leaving:' %}</p>
+                        <ul>
+                            <li>{% trans 'check for crowded places,' %}</li>
+                            <li>{% trans 'browse menus,' %}</li>
+                            <li>{% trans 'rate dishes,' %}</li>
+                            <li>{% trans 'find new places.' %}</li>
+                        </ul>
+                        <a href="/download/EatersLabBeta.apk" class="btn btn-primary btn-lg">{% trans 'Download for Android' %}</a>
+                    </div>
```

Comment:
not sure how to manage this but the link is /download/EatersLabBeta.apk on dev server and /download/EatersLab.apk on production server

Reply:
Can we modify files right before/after pushing to Heroku? We could modify `.env`

Reply:
It can bee a good idea, will remember to implement this when working with server files